README.md

```diff
@@ -49,7 +49,12 @@ It has two methods:
 
 ## `.parse(htmlString, options)`
 
-Takes a string of HTML and turns it into an AST, the only option you can currently pass is an object of registered `components` whose children will be ignored when generating the AST.
+Takes a string of HTML and turns it into an AST.
+
+### `options`
+
+* `components`: object of registered components whose children will be ignored when generating the AST. (defaults to `{}`)
+* `ignoreVoidElements`: boolean value to ignore void elements (defaults to `false`)
```

Comment:
Only thing I'd ask is to clarify what  the `ignoreVoidElements` option does. To me, the current description sounds like the library will ignore them and not include them in the AST. However, I believe it's actually that it will ignore the fact that the element is by definition a "void element" (according to the [`void-elements` NPM package](https://www.npmjs.com/package/void-elements) by way of the [HTML spec](http://www.w3.org/html/wg/drafts/html/master/syntax.html#void-elements)) and will instead allow there to be children inside of the "void element." Is that correct? Maybe something along these lines?
```suggestion
* `ignoreVoidElements`: boolean value to avoid treating void elements as void elements (i.e. they're allowed to have children) (defaults to `false`)
```

Reply:
Yeap, you are right.